spotty/spotty/Utils/Extensions/UIImageView+loadFromURL.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UIImageView+loadFromURL.swift
+//  spotty
+//
+//  Created by Ivaylo Kalaydzhiev on 16.06.22.
+//
+
+import UIKit
+
+extension UIImageView {
+    func loadFrom(URLAddress: String) {
+        guard let url = URL(string: URLAddress) else {
+            return
+        }
+        
+        DispatchQueue.main.async { [weak self] in
+            if let imageData = try? Data(contentsOf: url) {
```

Comment:
Refactor with guard let to avoid the pyramid of doom.

Reply:
done